com.vrcfury.vrcfury/Editor/VF/Feature/BakeHapticPlugsBuilder.cs

```diff
@@ -143,8 +143,18 @@ public class SpsRewriteToDo {
                     typeof(SkinnedMeshRenderer),
                     "material._SPS_Enabled"
                 );
-                var hapticsEnabledBinding = EditorCurveBinding.FloatCurve(
-                    pathToBake,
+                var hapticsSendersEnabledBinding = EditorCurveBinding.FloatCurve(
+                    pathToBake + "/Senders",
+                    typeof(GameObject),
+                    "m_IsActive"
+                );
+                var hapticsReceiversEnabledBinding = EditorCurveBinding.FloatCurve(
+                    pathToBake + "/Receivers",
+                    typeof(GameObject),
+                    "m_IsActive"
+                );
+                var legacyTipLightEnabledBinding = EditorCurveBinding.FloatCurve(
+                    pathToBake + "/LegacyDpsTip",
                     typeof(GameObject),
                     "m_IsActive"
                 );

```

Comment:
I think this is also missing /Animations (or maybe that was added after this pull request was open :( )

Reply:
Ah, good catch! I don't currently use depth animations so I just missed it when setting this up.

Pushed a commit that resolves this.